File: cpp/highest.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Enter the number of elements in the array: ";
    cin >> n;

    int numbers[n];
    cout << "Enter the elements of the array:" << endl;
    for (int i = 0; i < n; i++) {
        cin >> numbers[i];
    }

    int highest = numbers[0];
    for (int i = 1; i < n; i++) {
        if (numbers[i] > highest) {
            highest = numbers[i];
        }
    }

    cout << "Highest number in the array: " << highest << endl;
    return 0;
}

File: highest.java
import java.util.Scanner;

public class highest {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the number of elements in the array: ");
        int n = input.nextInt();

        int[] numbers = new int[n];
        System.out.println("Enter the elements of the array:");

        for (int i = 0; i < n; i++) {
            numbers[i] = input.nextInt();
        }

        int highest = numbers[0];

        for (int number : numbers) {
            if (number > highest) {
                highest = number;
            }
        }

        System.out.println("Highest number in the array: " + highest);
    }
}
